apps/cloud_composer/src/cloud_view.cpp

```diff
@@ -204,14 +228,18 @@ pcl::cloud_composer::CloudView::setAxisVisibility (bool visible)
   if (visible)
   {
     qDebug () << "Adding coordinate system!";
-    vis_->addOrientationMarkerWidgetAxes ( qvtk_->GetInteractor() );
+#if VTK_MAJOR_VERSION > 8
+    vis_->addOrientationMarkerWidgetAxes(qvtk_->interactor());
```

Comment:
Perhaps create a free function:
```cpp
auto get_interactor(typeof(qvtk_)& qvtk) {
#if VTK_MAJOR_VERSION > 8
  return qvtk.interactor();
#else
  return qvtk.GetInteractor();
}
```
This is used 3 times in this very file

Reply:
I have implemented this in the cloud_composer app - PTAL.